src/lang/zh-CN.json

```diff
@@ -453,6 +453,9 @@
     "WebHookUrl": "钉钉自定义机器人 Webhook 地址",
     "SecretKey": "钉钉自定义机器人加签密钥",
     "For safety, must use secret key": "出于安全考虑，必须使用加签密钥",
+    "isAtAll":"是否通知所有人",
+    "true":"是",
+    "false":"否",
```

Comment:
To prevent merge conflicts with weblate (more painfull than worth it), we only allow adding translation keys in `en.json`.
Please translate this after the PR is merged on weblate.kuma.pet

```suggestion
```